actions/datadog-release-metric/DatadogReleaseMetric.ts

```diff
@@ -8,6 +8,7 @@ import getVersion from "../utils/getVersion";
 
 const METRIC_TYPE = "count";
 const METRIC_NAME = "library.release.count";
+const PRE_RELEASE_SEPARATOR = "-";
```

Comment:
This looks good to me(assuming we are going x.x.x{-rc}y.y way).. Any reason we are not looking for '-rc' tag completely? I am thinking of any exception cases with the use of '-' -like doing any internal release for some reason in the future?

Reply:
This gives us flexibility in case we use a different release candidate identifier. From a semver perspective, a `-` represents a pre-release version and it should not be used for other purposes: https://semver.org/#spec-item-9